README.md

```diff
@@ -442,7 +465,7 @@ image tags. The tag convention is `{CML_VER}-dvc{DVC_VER}-base{BASE_VER}{-gpu}`:
 For example, `docker://dvcorg/cml:0-dvc2-base1-gpu`, or
 `docker://ghcr.io/iterative/cml:0-dvc2-base1`.
```

Comment:
btw is ghcr.io quicker on GH workflows? Should we promote it more?

Reply:
> _is ghcr.io quicker on GH workflows?_

I haven't made any benchmarks, but I'd say that it's a safe assumption to make.

> _Should we promote it more?_

Sure!